src/Ocelot.Provider.Kubernetes/Ocelot.Provider.Kubernetes.csproj

```diff
@@ -26,7 +26,6 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="KubeClient" Version="2.5.10" />
-    <PackageReference Include="KubeClient.Extensions.DependencyInjection" Version="2.5.10" />
```

Comment:
Why did you remove this pack reference?

Reply:
Because this fix propose IKubeApiClient to be registered in DI without helper from this package.

Reply:
Right, but if DI helpers don't meet requirements or cause negative consequences then custom helpers is appropriate alternative, IMO.

Reply:
In the next PR #2174 you've updated tte package version, and seems the pack is required. 😄

Reply:
Oh, my gosh! The pack version is outdated! 🤣  See [v2.5.12](https://www.nuget.org/packages/KubeClient.Extensions.DependencyInjection/2.5.12)

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/2180/commits/b8cd234cfaab3e436d92b3f751802f6fcffd0965

Reply:
Helper from there doesn't used anymore and not needed for next PR